qiskit/pulse/instructions/__init__.py

```diff
@@ -23,3 +23,4 @@
 """
 from .delay import Delay
 from .instruction import Instruction
+from .set_frequency import SetFrequency
```

Comment:
If we had a shift frequency instruction, would we combine them into one file? `from qiskit.pulse.instructions.frequency import ShiftFrequency, SetFrequency`? I don't have strong feelings but I kind of like that namespace

Reply:
`qiskit.pulse.instructions.frequency import ShiftFrequency, SetFrequency` looks appealing. A `ShiftFrequency` command could be a useful addition to simplify the interface.